prime-router/src/test/kotlin/azure/ActionHistoryTests.kt

```diff
@@ -562,11 +582,4 @@ class ActionHistoryTests {
         assertContains(blobUrls[0], org.receivers[0].fullName)
         assertContains(blobUrls[1], org.receivers[1].fullName)
     }
-
-    @Test
-    fun `test trimSchemaNameToMaxLength malformed URI`() {
-        val longMalformedURI = ":very_very:_long_name//with a badly formed URI that causes a parse exception"
```

Comment:
Do we still have/need handling of malformed URIs? If so, we should probably be keeping that part of the test at least.